internal/handlers/pg/pgdb/indexes.go

```diff
@@ -218,11 +226,22 @@ func createPgIndexIfNotExists(ctx context.Context, tx pgx.Tx, schema, table, ind
 	sql := `CREATE` + unique + ` INDEX IF NOT EXISTS ` + pgx.Identifier{index}.Sanitize() +
 		` ON ` + pgx.Identifier{schema, table}.Sanitize() + ` (` + strings.Join(fieldsDef, `, `) + `)`
 
-	if _, err = tx.Exec(ctx, sql); err != nil {
+	_, err = tx.Exec(ctx, sql)
```

Comment:
I see a bit of inconsistency: above you use `if err :=` and here you don't use it.
But it's not critical for me.